flash/core/model.py

```diff
@@ -809,7 +809,13 @@ def build_data_pipeline(
         if deserializer is None or type(deserializer) is Deserializer:
             deserializer = getattr(input_transform, "deserializer", deserializer)
 
-        data_pipeline = DataPipeline(input, input_transform, output_transform, deserializer, output)
+        data_pipeline = DataPipeline(
+            input=input,
+            input_transform=input_transform,
+            output_transform=output_transform,
+            deserializer=deserializer,
```

Comment:
Might be good to refer to it as `serve_input` or something instead of `deserializer`

Reply:
I think we would do this with the DataPipeline API change.